src/injected-content/scss/_theatermode.scss

```
@@ -110,4 +110,12 @@ b-channel-page-wrapper .stage:not(.aspect-16-9).me-video-stage{
     position: relative !important;
 }
 
+/* Co-Stream Theatermode top-bar fix*/
+.costream-rollout {
```

This is great! We want to make sure this only applies when theater mode is active though. We should just need to prepend this line with another parent css select to ensure `.costream-rollout` only gets this style if the parent `.stage` has the `.theaterMode` class.
Here is how you could do it:
```suggestion
.stage.theaterMode .costream-rollout {
```